Index: imx-uuc-1.0/linuxrc
===================================================================
--- imx-uuc-1.0.orig/linuxrc
+++ imx-uuc-1.0/linuxrc
@@ -129,7 +129,7 @@ function launch_uuc() {
 
 		mkdir functions/mass_storage.1
 		ln -s functions/mass_storage.1 configs/c.1/
-		echo /fat > functions/mass_storage.1/lun.0/file
+		echo /usr/share/imx-uuc/fat > functions/mass_storage.1/lun.0/file
 
 		ufb /dev/usb-utp$2/ep0 &
 
